hoot-core/src/main/cpp/hoot/core/conflate/highway/Roundabout.cpp

```diff
@@ -26,30 +26,34 @@
  */
 
 #include "Roundabout.h"
-#include <hoot/core/util/MapProjector.h>
-#include <hoot/core/io/OsmMapWriterFactory.h>
+#include <hoot/core/algorithms/linearreference/LocationOfPoint.h>
+#include <hoot/core/algorithms/splitter/WaySplitter.h>
+#include <hoot/core/elements/ElementConverter.h>
 #include <hoot/core/elements/NodeToWayMap.h>
 #include <hoot/core/index/OsmMapIndex.h>
-#include <hoot/core/ops/RemoveWayOp.h>
+#include <hoot/core/io/OsmMapWriterFactory.h>
 #include <hoot/core/ops/RemoveNodeOp.h>
-#include <hoot/core/elements/ElementConverter.h>
+#include <hoot/core/ops/RemoveWayOp.h>
+#include <hoot/core/ops/UnconnectedWaySnapper.h>
+#include <hoot/core/util/MapProjector.h>
+#include <hoot/core/visitors/FindWaysVisitor.h>
+
 #include <geos/geom/Geometry.h>
 #include <geos/geom/CoordinateSequence.h>
 #include <geos/geom/LineString.h>
-#include <hoot/core/algorithms/splitter/WaySplitter.h>
-#include <hoot/core/algorithms/linearreference/LocationOfPoint.h>
 
 namespace hoot
 {
 
 typedef boost::shared_ptr<geos::geom::Geometry> GeomPtr;
 
-Roundabout::Roundabout():
-  _status(Status::Invalid)
+Roundabout::Roundabout()
+  :  _status(Status::Invalid),
+    _overrideStatus(false)
```

Comment:
`_overrideStatus` is for situations were the roundabout was found in the secondary dataset but should still be transferred to the conflated output.